_internal/DeerLister.php

```diff
@@ -269,16 +306,27 @@ public function render(string $directory): string
                 {
                     $title = $parsedown->getTitle();
                 }
-                break;
+            }
+
+            if (!$displayBack && $f["name"] === "..")
+            { }
+            else if (!$displayOthers && isset($displayMode) && !$this->fileDisplays[$displayMode]->doesHandle($f["extension"]))
+            { }
+            else
```

Comment:
What are these empty blocks

Reply:
It's the cases where the file shouldn't be displayed, I think it's cleaner that just doing a huge if that negate all of those